### ISSUE TITLE
Nice script. Looking good and promising.

### ISSUE THREAD

**Issue by axisdos** [pinodex/boardy#1]
Hey man, good job on creating this forum script. I think it look nice and very promising. Any new features or improvements coming?

**Comment by pinodex** [comment]
Yup. I'm planning to make this in active development. Thanks anyway!
